src/mainwin.cc

```diff
@@ -2967,13 +2967,21 @@ QString MainWindow::exportPath(const char *suffix) {
 	QString path;
 	auto path_it = this->export_paths.find(suffix);
 	if(path_it != export_paths.end())
-		path = path_it->second;
+	{
+		path = QFileInfo(path_it->second).absolutePath();
+		if(activeEditor->filepath.isEmpty())
+			path += QString(_("/Untitled")) + suffix;
+		else
+			path += QString(_("/")) + QFileInfo(activeEditor->filepath).completeBaseName() + suffix;
```

Comment:
Not newly introduced with the change, but maybe we can drop the translation marker from `"/"` (here and in the other branch too) as this is not language dependent.